src/processors/ProcessorStore.cpp

```diff
@@ -210,5 +214,12 @@ void ProcessorStore::createProcReplaceList (PopupMenu& menu, int& menuID, BasePr
                    && procInfo.numOutputs == procToReplace->getNumOutputs()
                    && name != procToReplace->getName();
         },
-        procToReplace);
+        procToReplace,
+        nullptr,
+        nullptr);
+}
+
+void ProcessorStore::createProcFromCableClickList (PopupMenu& menu, int& menuID, BaseProcessor* cableClickStartProc, BaseProcessor* cableClickEndProc) const
+{
+    createProcListUnfiltered (*this, menu, menuID, nullptr, cableClickStartProc, cableClickEndProc);
```

Comment:
If we did want to enforce that connections should only be allowed to be replaced with 1->1 modules, this would be the place to do it.

Reply:
Good idea, with the new logic this is necessary.